ethereum/core/src/main/java/org/hyperledger/besu/ethereum/worldstate/WorldStateArchive.java

```diff
@@ -30,11 +30,11 @@
 public interface WorldStateArchive {
   Hash EMPTY_ROOT_HASH = Hash.wrap(MerklePatriciaTrie.EMPTY_TRIE_NODE_HASH);
 
-  Optional<WorldState> get(final Hash rootHash);
+  Optional<WorldState> get(final Hash rootHash, Hash blockHash);
 
-  boolean isWorldStateAvailable(final Hash rootHash);
+  boolean isWorldStateAvailable(final Hash rootHash, Hash blockHash);
```

Comment:
missing final for the blockHash field

Reply:
For interfaces final on parameters has no impact (much like the public and abstract modifiers).  Rather than add additional finals I removed the ones that were not there.  A mix of final and non-final params is confusing.